functions/validateAddressWithShipEngine/main.js

```diff
@@ -0,0 +1,19 @@
+const functions = require('firebase-functions');
+const mapDataAccordingToSchema = require('./helpers/mapDataAccordingToSchema');
+const validateAddress = require('./services/validateAddress');
+const config = require('./services/getConfig');
+
+const validateAddressWithShipEngine = async (data, context) => {
+    if (!context.auth) {
+        throw new functions.https.HttpsError(
+            'unauthenticated',
+            'This function requires authentication.',
+        );
+    }
```

Comment:
all callables functions should not check for context.auth instead to do this the user should use the isAuth policy, also add this case to documentation